### PR TITLE
Show default values in CompositeEndpointDataSource's debugger display…

### PR DESCRIPTION
**Pull request by kichalla**
… string

### CODE REVIEW

File: src/Microsoft.AspNetCore.Routing/CompositeEndpointDataSource.cs [hunk 1]
```diff
@@ -127,9 +127,11 @@ private string DebuggerDisplayString
                         var template = matcherEndpoint.RoutePattern.RawText;
                         template = string.IsNullOrEmpty(template) ? "\"\"" : template;
                         sb.Append(template);
-                        var requiredValues = matcherEndpoint.RequiredValues.Select(kvp => $"{kvp.Key} = \"{kvp.Value ?? "null"}\"");
                         sb.Append(", Required Values: new { ");
-                        sb.Append(string.Join(", ", requiredValues));
+                        sb.Append(string.Join(", ", GetValues(matcherEndpoint.RequiredValues)));
```
Comment (kichalla) [inline src/Microsoft.AspNetCore.Routing/CompositeEndpointDataSource.cs]: Sample output of `MvcSandbox` app:
```
"", Required Values: new { action = "Index", controller = "Home", page = null, area = null }, Defaults: new { controller = "Home", action = "Index", page = null, area = null }, Order:1
Home, Required Values: new { action = "Index", controller = "Home", page = null, area = null }, Defaults: new { controller = "Home", action = "Index", page = null, area = null }, Order:2
Home/Index/{id?}, Required Values: new { action = "Index", controller = "Home", page = null, area = null }, Defaults: new { controller = "Home", action = "Index", page = null, area = null }, Order:3
Admin/Index, Required Values: new { page = "/Index", area = "Admin", action = null, controller = null }, Defaults: new { page = "/Index", area = "Admin", action = null, controller = null }, Order:0
Admin, Required Values: new { page = "/Index", area = "Admin", action = null, controller = null }, Defaults: new { page = "/Index", area = "Admin", action = null, controller = null }, Order:0
PagesHome, Required Values: new { page = "/PagesHome", action = null, controller = null, area = null }, Defaults: new { page = "/PagesHome", action = null, controller = null, area = null }, Order:0

```

File: src/Microsoft.AspNetCore.Routing/CompositeEndpointDataSource.cs [hunk 2]
```diff
@@ -150,6 +156,20 @@ private string DebuggerDisplayString
                     }
                 }
                 return sb.ToString();
+
+                IEnumerable<string> GetValues(IEnumerable<KeyValuePair<string, object>> values)
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Routing/CompositeEndpointDataSource.cs]: Should we maybe call this `FormatValues`?

**Review by rynowak — APPROVED** [review]

**Comment by rynowak** [comment]
@kichalla - I accidentally merged this 😞 - sorry I had too many windows open.

**Comment by kichalla** [comment]
No worries! I will rename the method though

**Pull request merged**